src/css/custom.css

```
@@ -15,6 +15,11 @@
 @import './font.css';
 @import './types.css';
 
+html:has(.theme-doc-markdown) {
+  scroll-behavior: smooth;
+}
+.container {
+}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)